qiskit/chemistry/qmolecule.py

```diff
@@ -14,6 +14,7 @@
 
 """ QMolecule """
```

Comment:
```suggestion
"""The QMolecule module."""
```

Reply:
I think the QMolecule class requires a general update of all documentation and type hints.
I am not sure though whether this should be part of this PR.

Reply:
There are any number of files that could do with being updated to typehints. This can wait until later